extensions/binding/org.eclipse.smarthome.binding.digitalstrom/src/main/java/org/eclipse/smarthome/binding/digitalstrom/handler/BridgeHandler.java

```diff
@@ -0,0 +1,643 @@
+        String[] loginConfig;
+
+        public InitalizazeRunnable(BridgeHandler bridge, Configuration configuration, String[] loginConfig) {
+            this.bridge = bridge;
+            this.configuration = configuration;
+            this.loginConfig = loginConfig;
+        }
+
+        @Override
+        public void run() {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_PENDING, "Loading configurations");
+            // get Configurations
+            if (configuration.get(DigitalSTROMBindingConstants.SENSOR_DATA_UPDATE_INTERVAL) != null
+                    && !configuration.get(DigitalSTROMBindingConstants.SENSOR_DATA_UPDATE_INTERVAL).toString().trim()
+                            .replace(" ", "").isEmpty()) {
+
+                Config.SENSORDATA_REFRESH_INTERVAL = Integer.parseInt(
+                        configuration.get(DigitalSTROMBindingConstants.SENSOR_DATA_UPDATE_INTERVAL).toString() + "000");
+            }
+            if (configuration.get(DigitalSTROMBindingConstants.TOTAL_POWER_UPDATE_INTERVAL) != null
+                    && !configuration.get(DigitalSTROMBindingConstants.TOTAL_POWER_UPDATE_INTERVAL).toString().trim()
+                            .replace(" ", "").isEmpty()) {
+
+                Config.TOTAL_POWER_UPDATE_INTERVALL = Integer.parseInt(
+                        configuration.get(DigitalSTROMBindingConstants.TOTAL_POWER_UPDATE_INTERVAL).toString() + "000");
+            }
+
+            if (configuration.get(DigitalSTROMBindingConstants.SENSOR_WAIT_TIME) != null && !configuration
+                    .get(DigitalSTROMBindingConstants.SENSOR_WAIT_TIME).toString().trim().replace(" ", "").isEmpty()) {
+
+                Config.SENSOR_READING_WAIT_TIME = Integer
+                        .parseInt(configuration.get(DigitalSTROMBindingConstants.SENSOR_WAIT_TIME).toString() + "000");
+            }
+
+            if (configuration.get(DigitalSTROMBindingConstants.DEFAULT_TRASH_DEVICE_DELEATE_TIME_KEY) != null
+                    && !configuration.get(DigitalSTROMBindingConstants.DEFAULT_TRASH_DEVICE_DELEATE_TIME_KEY).toString()
+                            .trim().replace(" ", "").isEmpty()) {
+
+                Config.TRASH_DEVICE_DELEATE_TIME = Integer.parseInt(configuration
+                        .get(DigitalSTROMBindingConstants.DEFAULT_TRASH_DEVICE_DELEATE_TIME_KEY).toString());
+            }
+            if (configuration.get(DigitalSTROMBindingConstants.TRUST_CERT_PATH_KEY) != null
+                    && !configuration.get(DigitalSTROMBindingConstants.TRUST_CERT_PATH_KEY).toString().trim()
+                            .replace(" ", "").isEmpty()) {
+
+                Config.TRUST_CERT_PATH = configuration.get(DigitalSTROMBindingConstants.TRUST_CERT_PATH_KEY).toString();
+            }
+
+            logger.debug("Initializing DigitalSTROM Manager.");
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_PENDING, "Checking connection");
```

Comment:
again, better remove the statusUpdate here. It is not meant as a progress monitor, but rather only for updating when you are done with processing.